operator/README.md

```diff
@@ -255,7 +255,7 @@ make kuttl
 
 These make targets will add the executable to your `$GOPATH`.
 If that is not on your `$PATH`, then you can install the Operator SDK from its [release page](https://github.com/operator-framework/operator-sdk/releases)
-and kuttl from its [release page](https://github.com/kudobuilder/kuttl/releases/tag/v0.15.0).
+and kuttl from its [release page](https://github.com/porridge/kuttl/releases).
```

Comment:
Why this change?

Reply:
See PR description:

> Use a modified kuttl with support for conditional inclusion of files, see https://github.com/kudobuilder/kuttl/pull/483

That change was merged upstream by now, but it might still be a while until we see a new release.